atlantic-2/proposals/03-03-2022-decrease-vp-threshold.json

```diff
@@ -0,0 +1,18 @@
+{
+    "title": "Change Max Voting PowerEnforcement",
+    "description": "Decrese threshold to start enforcing threshold soon",
+    "changes": [
+      {
+        "subspace": "staking",
+        "key": "MaxVotingPower",
```

Comment:
is this key supposed to be MaxVotingPowerRatio?

Reply:
no... https://github.com/sei-protocol/sei-cosmos/blob/main/x/staking/types/params.go#L86

Reply:
ok sounds good, just verifying since in the queried params it outputs `ratio`

Reply:
yeah, i dont like it tbh. i thought it would be MaxPowerRatio at first too